Resources/Prototypes/Roles/Jobs/Command/blue_shield.yml

```diff
@@ -0,0 +1,43 @@
+- type: job
+  id: Blueshield
+  name: blueshield
+  weight: 20
+  startingGear: BlueshieldGear
+  departments:
+  - Command
+  icon: "BlueShield"
+  requireAdminNotify: true
+  joinNotifyCrew: true
+  supervisors: job-supervisors-centcom
+  canBeAntag: false
+  access:
+  - BlueShield
+  - Cargo
+  - Salvage
+  - Command
+  - Engineering
+  - Atmospherics
+  - External
+  - Maintenance
+  - Medical
+  - Research
+  - Security
+  - Brig
+  - Service

```

Comment:
this much access is awful. You are basically giving a non-head free roundstart AA. I get that the idea is to be captain's guard, but captain already has AA. they shouldn't have anywhere near this much access.

Reply:
Not the captain, but the heads, all the heads, damn it.